### PR TITLE
unknown events will be catched and no longer break the whole client

### PR DESCRIPTION
**Pull request by Phyrone**

### CODE REVIEW

File: src/main/java/com/github/manevolent/ts3j/protocol/socket/client/LocalTeamspeakClientSocket.java [hunk 1]
```diff
@@ -433,9 +433,13 @@ public void process(AbstractTeamspeakClientSocket client,
         if (processor != null) {
             processor.process(client, singleCommand);
         } else if (singleCommand.getName().startsWith("notify")) {
-            TS3Event event = TS3Event.createEvent(singleCommand);
+            try {
+                TS3Event event = TS3Event.createEvent(singleCommand);
+                commandExecutionService.submit(() -> listeners.forEach(event::fire));
+            } catch (IllegalArgumentException exception) {
+                exception.printStackTrace();
```
Comment (Manevolent) [inline src/main/java/com/github/manevolent/ts3j/protocol/socket/client/LocalTeamspeakClientSocket.java:440]: Can we propagate this exception to an exception listener?  I.e.,

`getExceptionHandler().accept(exception);`

Granted, I don't do this everywhere right now, but these especially seem like we should propagate them forward and notify the user if possible.

Comment (Phyrone) [reply src/main/java/com/github/manevolent/ts3j/protocol/socket/client/LocalTeamspeakClientSocket.java:440]: do you mean something like an event for and unknow teamspeak event?

Comment (Manevolent) [reply src/main/java/com/github/manevolent/ts3j/protocol/socket/client/LocalTeamspeakClientSocket.java:440]: That would also be a great solution :)

Comment (Phyrone) [reply src/main/java/com/github/manevolent/ts3j/protocol/socket/client/LocalTeamspeakClientSocket.java]: done

**Comment by Phyrone** [comment]
here it is

**Comment by Manevolent** [comment]
Looks awesome!  Thank-you! :)

**Pull request merged**